packages/logger/tests/unit/logBuffer.test.ts

```diff
@@ -0,0 +1,123 @@
+import { describe, expect, it, vi } from 'vitest';
+import { CircularMap, SizedItem, SizedSet } from '../../src/logBuffer';
+
+describe('SizedItem', () => {
+  it('should calculate the byteSize based on JSON serialization', () => {
+    // Prepare
+    const logEntry = { message: 'hello world' };
+    // Act
+    const item = new SizedItem(logEntry, 1);
+    // Assess
+    const expectedByteSize = Buffer.byteLength(JSON.stringify(logEntry));
+    expect(item.byteSize).toBe(expectedByteSize);
+  });
+});
+
+describe('SizedSet', () => {
+  it('should add an item and update currentBytesSize correctly', () => {
+    // Prepare
+    const set = new SizedSet<object>();
+    const item = new SizedItem({ test: 'value' }, 1);
+    // Act
+    set.add(item);
+    // Assess
+    expect(set.currentBytesSize).toBe(item.byteSize);
+    expect(set.has(item)).toBe(true);
+  });
+
+  it('should delete an item and update currentBytesSize correctly', () => {
+    // Prepare
```

Comment:
In all test cases, let's add space between each section, i.e. add a new line before each `// Act` and `// Assess`